sqlalchemy_redshift/dialect.py

```diff
@@ -328,9 +331,13 @@ def get_column_specification(self, column, **kwargs):
 
     def _fetch_redshift_column_attributes(self, column):
         text = ""
-        if not hasattr(column, 'info'):
-            return text
-        info = column.info
+        if sa.__version__ >= '1.3.0':
+            info = column.dialect_options['redshift']
```

Comment:
If a user is on SQLAlchemcy 1.3.0+, but passes `info`, will it still show up in dialect_options? Or do we need to check for an `info` attr to handle that case and maintain compatibility with existing usage of this library?

Reply:
No, info won't show up on 1.3.0+ and my best suggestion would be to explicitly update the docs to force the new syntax. Otherwise, there's a risk of mixing old and new syntax and related inconsistencies. This may be a good ground to bump version to 0.8.x

Reply:
cc @graingert for awareness on the potential need for bumping the baseline SQA version and major version.

Reply:
So info will still work for SA <1.3 users? Update the docs to drop the `info={` but keep the minimum sa version as-is in setup.py

Reply:
@graingert , that's what I did

Reply:
> So info will still work for SA <1.3 users? Update the docs to drop the info={ but keep the minimum sa version as-is in setup.py

I think I'm fine with this as long as the docs include some commentary about the need to substitute `info={...` if using SA<1.3.